src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppAppRunner/AspNetAppAppRunner.csproj

```diff
@@ -24,9 +24,8 @@
 
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
-    <PackageReference Include="Amazon.CDK" Version="1.128.0" />
-    <PackageReference Include="Amazon.CDK.AWS.AppRunner" Version="1.128.0" />
-    <PackageReference Include="Amazon.CDK.AWS.ECS.Patterns" Version="1.128.0" />
+    <PackageReference Include="Amazon.CDK.Lib" Version="2.13.0" />
+    <PackageReference Include="Constructs" Version="10.0.70" />
```

Comment:
Since `Amazon.CDK.Lib` has a dependency on `Constructs` you don't need to add that `PackageReference`.

Reply:
Done